src/compile/data/aggregate.ts

```diff
@@ -155,11 +156,15 @@ export class AggregateNode extends DataFlowNode {
   }
 
   public producedFields() {
-    const out = {};
+    let out = {};
 
     for (const field of keys(this.measures)) {
       for (const op of keys(this.measures[field])) {
-        out[this.measures[field][op] || `${op}_${field}`] = true;
+        if (keys(this.measures[field][op]).length === 0) {
+          out[`${op}_${field}`] = true;
+        } else {
+          out = {...out, ...this.measures[field][op]};
```

Comment:
`Object.assign(out, ...this.measures[][])`  or just `assign`

Reply:
I believe `Object.assign` is exclusive to ES6. The only reference to assign I could find was `__assign(...)` from tslib which we probably shouldn't be using (?).